From 119992bb874b6b2c78b515bde12d3fd5e9bdb63a Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 06 Nov 2019 21:37:28 +0200
Subject: [PATCH] [SW-0] package: update preboot file with ARCH prefix

This is since we got more then one flavour of preboot in Goya2

Change-Id: I0f122aac4d09cb8f5d6ec7477a80da0dd5b7466e
---

diff --git a/package/btl/btl.mk b/package/btl/btl.mk
index 65bf98a..2eab705 100644
--- a/package/btl/btl.mk
+++ b/package/btl/btl.mk
@@ -5,7 +5,7 @@
 BTL_DEPENDENCIES = preboot
 
 define BTL_BUILD_CMDS
-	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) all
+	$(TARGET_MAKE_ENV) $(MAKE) ARCH=$(ARCH) -C $(@D) all
 endef
 
 BTL_INSTALL_IMAGES = YES
diff --git a/package/preboot/preboot.mk b/package/preboot/preboot.mk
index 44361ba..8c2a83d 100644
--- a/package/preboot/preboot.mk
+++ b/package/preboot/preboot.mk
@@ -8,7 +8,7 @@
 
 PREBOOT_INSTALL_IMAGES = YES
 define PREBOOT_INSTALL_IMAGES_CMDS
-	cp -dpf $(@D)/preboot.bin $(BINARIES_DIR)/
+	cp -dpf $(@D)/preboot.bin $(BINARIES_DIR)/$(ARCH)-preboot.bin
 	cp -dpf $(@D)/common/pcie_phy_fw.bin $(BINARIES_DIR)/
 endef
 
